src/main/java/org/ibp/api/java/impl/middleware/germplasm/GermplasmNameService.java

```diff
@@ -10,4 +15,7 @@
 
 	public Integer createName(String programUUID, GermplasmNameRequestDto germplasmNameRequestDto, Integer gid);
 
+	public List<GermplasmGroupNamingResult> createCodeNames(GermplasmCodeNameBatchRequestDto germplasmCodeNameBatchRequestDto);
```

Comment:
Is GermplasmGroupNaming the proper name? I would think by that name this is related to group germplasm actions. Maybe rename as GermplasmCodingResult?

Reply:
Done